internal/cli/backup/restores/start.go

```diff
@@ -60,14 +60,37 @@ func (opts *StartOpts) initStore(ctx context.Context) func() error {
 var startTemplate = "Restore job '{{.Id}}' successfully started\n"
 
 func (opts *StartOpts) Run() error {
+	r, err := opts.store.CreateRestoreFlexClusterJobs(opts.ConfigProjectID(), opts.clusterName, opts.newFlexBackupRestoreJobCreate())
```

Comment:
[q] Will this work for non-flex clusters?

Reply:
no, it will return `cannotUseNotFlexWithFlexApisErrorCode`. The lines after this one make sure that we run the cluster dedicated function (`CreateRestoreJobs`) in the event we got the error  `cannotUseNotFlexWithFlexApisErrorCode`

Reply:
Oh, I ready to quickly.

I'm so used to reading `if err != nil` under something that can fail I failed to see that it was `==` instead of `!=`